tests/unittests/OperatorTest.cpp

```diff
@@ -5063,6 +5063,94 @@ TEST_P(InterpOnly, Modulo2) {
   }
 }
 
+/// Test a DotProduct operator with 1D inputs.
+TEST_P(InterpAndCPU, dotProduct1D) {
```

Comment:
That should work on OpenCL backend as well it's just a MulNode. Any reason it's not included here?